packages/passport-interface/src/api/requestIssuedPCDs.ts

```diff
@@ -1,26 +1,24 @@
-import { FeedRequest, FeedResponse } from "@pcd/passport-interface";
+import { FeedRequest, FeedResponse } from "../RequestTypes";
+import { POST } from "./constants";
 
 /**
  * Given the information the server knows about the user, it is able to
  * 'issue' some pcds to the user. This function requests the set of PCDs
  * that the server wants to 'issue' to the user.
+ *
+ * Never rejects. All information encoded in the resolved response.
  */
 export async function requestIssuedPCDs(
```

Comment:
Since the addition of generic issuance, we don't actually use this anywhere, so it can be deleted.

Reply:
thanks - deleted